test/compose/uptwice_idempotent/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+services:
+  app:
+    image: alpine
+    command: [sleep, infinity]
+    networks:
+      - net1
+networks:
+  net1:
```

Comment:
compose creates and used custom networks by default, what is the point to declare one here?

Reply:
Good point.